src/js/Features.js

```diff
@@ -130,58 +130,50 @@ Features.prototype.isEnabled = function (featureName) {
 };
 
 Features.prototype.generateElements = function (featuresElements) {
-    var self = this;
+    const self = this;
 
     self._featureChanges = {};
 
-    var listElements = [];
+    const listElements = [];
+
+    for (let i = 0; i < self._features.length; i++) {
+        let feature_tip_html = '';
+        const rawFeatureName = self._features[i].name;
+        const featureBit = self._features[i].bit;
 
-    for (var i = 0; i < self._features.length; i++) {
-        var feature_tip_html = '';
         if (self._features[i].haveTip) {
-            feature_tip_html = '<div class="helpicon cf_tip" i18n_title="feature' + self._features[i].name + 'Tip"></div>';
+            feature_tip_html = `<div class="helpicon cf_tip" i18n_title="feature${rawFeatureName}Tip"></div>`;
         }
 
-        var newElements = [];
+        const newElements = [];
+
         if (self._features[i].mode === 'select') {
             if (listElements.length === 0) {
-                newElements.push($('<option class="feature" '
-                    + 'value="-1" '
-                    + 'i18n="featureNone" />'));
+                newElements.push($('<option class="feature" value="-1" i18n="featureNone" />'));
             }
-
-            var newElement = $('<option class="feature" id="feature-'
-                + i
-                + '" name="'
-                + self._features[i].name
-                + '" value="'
-                + self._features[i].bit
-                + '" i18n="feature' + self._features[i].name + '" />');
+            const newElement = $(` <option class="feature" id="feature-${i}" name="${rawFeatureName}" value="${featureBit}" i18n="feature${rawFeatureName}" />`);
```

Comment:
Stray space at the start of the string. :space_invader:

Reply:
👽